packages/ia-components/sandbox/audio-players/youtube-player.jsx

```diff
@@ -53,8 +53,12 @@ class YoutubePlayer extends Component {
    * Update component only if there is a change in video id
    * If above update video id and selected track
    */
-  shouldComponentUpdate(nextProps) {
-    const { id } = this.state;
+  shouldComponentUpdate(nextProps, nextState) {
```

Comment:
The comment above this method indicates that it processes to the end only if the video ID has changed, however it looks like it's only if the player prop has changed. Is that still correct?

Reply:
it still stands. thanks for checking